exercises/sum.cpp

```diff
@@ -1,8 +1,19 @@
-/*
-  Write a program that takes as input two numbers and print the sum.
-
-  Output:
-  Insert the first number: 1
-  Insert the second number: 2
-  Sum: 3
-*/
+#include <iostream>
+using namespace std;
+
+template <typename T, typename X>
+
+T somma(T a, X b) {
+	return a + b;
+}
+
+int main() {
+	int a, b;
+	cout << "Inserire primo addendo: " << endl;
+	cin >> a;
+	cout << "Inserire secondo addendo: " << endl;
+	cin >> b;
+	cout << "La somma di " << a << " e " << b << " e' pari a: " << somma(a, b);
+	return 0;
```

Comment:
potresti sostituire 0 con EXIT_SUCCESS in modo da migliorare la leggibilità del codice
```c++
#define EXIT_SUCCESS 0
...
return EXIT_SUCCESS;
```